with-dev-client/android/gradle.properties

```
@@ -26,4 +26,4 @@ android.useAndroidX=true
 android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.54.0
+FLIPPER_VERSION=0.54.0
```

```suggestion
FLIPPER_VERSION=0.75.1
```
Shouldn't it be the same as on iOS, should it?

---

We should probably make this change in [expo-template-bare-minimum](https://github.com/expo/expo/tree/master/templates/expo-template-bare-minimum), however we'll need to test that it works first.